resources/sumo_exporter/crossroad.py

```diff
@@ -24,25 +24,20 @@ def __init__(self, crossroadType):
 
     def init_from_wbt_string(self, wbtString):
         """Extract info from the wbtString matching the node."""
-        try:
-            self.id = re.findall(r'id\s*"([^"]*)"', wbtString)[0]
-        except:
-            pass
-        try:
-            self.translation = [float(x) for x in re.findall(r'translation\s*(%s\s*%s\s*%s)' %
-                                                             (floatRE, floatRE, floatRE), wbtString)[0].split()]
-        except:
-            pass
-        try:
-            self.rotation = [float(x) for x in re.findall(r'rotation\s*(%s\s*%s\s*%s\s*%s)' %
-                                                          (floatRE, floatRE, floatRE, floatRE), wbtString)[0].split()]
-        except:
-            self.rotation = [0.0, 0.0, 1.0, 0.0]
-        try:
-            self.connectedRoadIDs = [x.replace('"', '') for x in re.findall(r'connectedRoadIDs\s*\[([^\]]*)\]',
-                                                                            wbtString)[0].split()]
-        except:
-            pass
+
+        id = re.findall(r'id\s*"([^"]*)"', wbtString)
+        self.id = id[0] if not id else self.id
```

Comment:
This seems wrong, shouldn't be
```suggestion
        self.id = id[0] if id else self.id
```

The same wrong conditional is also used in the following lines for `translation`, `rotation`, `connectedRoadIDs`, `roadNumber` and `roadsWidth`.
And in the changes to `resources/sumo_exporter/road.py` file aiming at removing the `try/except` blocks.

Reply:
yes definitely, got it backwards :cry:

Reply:
ops, you're right I got it backwards